spec/support/request/admin_helper.rb

```diff
@@ -3,13 +3,16 @@
 module AdminHelper
   def toggle_columns(*labels)
     # open dropdown
-    find("div#columns-dropdown", text: "COLUMNS").click
+    # case insensitive search for "Columns" text
+    find("div#columns-dropdown", text: /columns/i).click
 
-    labels.each do |label|
-      find("div#columns-dropdown div.menu div.menu_item", text: label).click
+    within "div#columns-dropdown" do
+      labels.each do |label|
+        find("div.menu div.menu_item", text: /#{label}/i).click
```

Comment:
I was uncomfortable about sending regexps as strings (eg `"^.{0,1}Producer$"`), so I figured out a way to accept either string or regexp here. This way, strings are converted to case-insensitive regexp, but a regexp is passed through directly.

```suggestion
        # Convert label to case-insensitive regexp if not one already
        label = /#{label}/i unless label.is_a?(Regexp)

        find("div.menu div.menu_item", text: label).click
```

I considered also modifying the regexp to make it case-insensitive but decided it would be better not to modify it. Clearly, I thought way too much about it..

So in the spec, we would need to add the 'i', for example:

```diff
- toggle_columns /^.{0,1}Producer$/
+ toggle_columns /^.{0,1}Producer$/i